docs: Restructure tools and Capella documentation (#1034)
DSD-DBS/capella-collab-manager: docs/user/docs/index.md

Welcome onboard of the Modeling Collaboration Manager. This platform helps you
co-work on architectures using tools like
[Capella](https://www.eclipse.org/capella/),
[Papyrus](https://www.eclipse.org/papyrus/),
[pure::variants](https://www.pure-systems.com/purevariants) and
[Jupyter](https://jupyter.org/) in combination with
[py-capellambse](https://github.com/DSD-DBS/py-capellambse).

This platform wraps the modeling tools to create an environment where users can
can work directly in the project context without having to install or configure
the tool. It also takes care of the housekeeping of the modeling projects

> themselves. And that's not all: We also offer project access management and the
> site is completely responsive - and thus also usable on smartphones.

Reviewer: ```suggestion
themselves. And that's not all: We also offer project access management and since the site is completely responsive it is also usable on smartphones.
```

## Automations and model-derived products

> As you can see, Git is quite in the middle of the modeling lifecycle. We also

Reviewer: ```suggestion
Git is quite in the middle of the modeling lifecycle. We also
```
Do we still see that if the section above is deleted?

stream that connection to your browser. We call instances of those containers
**Sessions**. To make this platform scalable and allow people co-work

> comfortable we use [Kubernetes](https://kubernetes.io/) to run the **Session**

Reviewer: ```suggestion
comfortably we use [Kubernetes](https://kubernetes.io/) to run the **Session**
```

- **Model manager** - can do same as both users above but also can invite or
  remove users from the managed projects and control their access rights.

You can find more details about
[how to request a session here](sessions/request.md).

> 

Reviewer: is this still linked somewhere?

Author: There a few broken links, but I fixed them all.
